src/app/hooks/useViewTracker/index.jsx

```diff
@@ -72,7 +72,16 @@ const useViewTracker = (props = {}) => {
 
         if (shouldSendEvent) {
           if (optimizely) {
-            optimizely.track('component_views');
+            const overrideAttributes = {
```

Comment:
Don't see any tests covering the override attributes. Is it worth writing/modifying the expected to see if it's called with the correct overrides?